spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/util/JacksonUtils.java

```diff
@@ -84,4 +85,13 @@ public static Map<String, String> deserialize2Map(String jsonStr) {
 			throw new RuntimeException("Json to map failed.", e);
 		}
 	}
+
+	public static Map<String, Object> json2Map(String jsonStr) {
```

Comment:
Using the `deserialize2Map` method instead of `json2Map` method , without adding a method with duplicate functionality.